firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/cobs.txt

```diff
@@ -0,0 +1,131 @@
+Scenario: The cobs::decode method correctly decodes encoded buffers
```

Comment:
Since COBS.h defines the encode function before the decode function, the encode tests should come before the decode tests. Also, these are functions and not methods (as reflected in the GIVEN), so the scenario description should be consistent with that.